drydock/templates/drydock/k8s/ingress/static-cache.yml

```diff
@@ -0,0 +1,33 @@
+{%- for service, config in DRYDOCK_NGINX_STATIC_CACHE_CONFIG.items() %}
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: ingress-static-{{ service }}
+  namespace: {{ K8S_NAMESPACE }}
+  annotations:
+    nginx.ingress.kubernetes.io/proxy-body-size: 8m
+    nginx.ingress.kubernetes.io/proxy-buffering: "on"
+    nginx.ingress.kubernetes.io/configuration-snippet: |
+      # Cache settings
+      proxy_cache_valid 404 10m;
+      proxy_cache_use_stale error timeout updating http_404 http_500 http_502 http_503 http_504;
+      proxy_cache static-cache;
+      proxy_cache_valid any 120m;
+      proxy_cache_bypass $http_x_purge;
+      add_header X-Cache-Status $upstream_cache_status;
+      {{ patch("static-cache-config") | indent(6)}}
```

Comment:
Does nginx play nicely with overrides for this o does it complain if you try to configure the same value twice?

For example you set `proxy_cache_valid any 300m;` in the patch.

Reply:
I've tested this, but that should be the expected behavior

Reply:
Tested on musashi.chennai. Working as expected